app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -489,6 +483,22 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>(), CancelPledge
         finish()
     }
 
+    private fun pledgeFragment() = supportFragmentManager
+            .findFragmentByTag(PledgeFragment::class.java.simpleName) as PledgeFragment?
+
+    private fun pledgeFragmentShouldAnimateOut(): Boolean {
+        val backStackEntryCount = supportFragmentManager.backStackEntryCount
+        val backStackIsNotEmpty = backStackEntryCount > 0
+        val topOfStackIndex = backStackEntryCount.minus(1)
```

Comment:
still find it so interesting that the `-` operator and the function call are seemingly interchangable in Kotlin